app/addons/databases/resources.js

```diff
@@ -137,6 +137,8 @@ function(app, FauxtonAPI, Documents) {
       var updateSeq = this.get("update_seq");
       if (full || (typeof(updateSeq) === 'number')) {
         return updateSeq;
+      } else if (typeof(updateSeq) === 'string') {
+        return updateSeq.split('-')[0];
```

Comment:
Above you return a number as like as below. Here you return a string. Wouldn't it cause any problems somewhere ahead?